doc/northboundinteractions.md

```diff
@@ -1,4 +1,4 @@
-# Northbound NGSI Interactions
+# Traffic North of the IoT Agent -  NGSI Interactions
```

Comment:
In line 60:

> South Bound protocols (typically lightweight protocols aimed to constrained devices)

Reply:
In line 65:

> Devices, in the South Bound

Reply:
fixed  376c3e2 -  Replaced with *traffic sent and received on the South Port*